src/main/java/org/apache/ibatis/parsing/XNode.java

```diff
@@ -130,71 +130,47 @@ public String getStringBody() {
   }
 
   public String getStringBody(String def) {
-    if (body == null) {
-      return def;
-    } else {
-      return body;
-    }
+    return body == null ? def : body;
   }
 
   public Boolean getBooleanBody() {
     return getBooleanBody(null);
   }
 
   public Boolean getBooleanBody(Boolean def) {
-    if (body == null) {
-      return def;
-    } else {
-      return Boolean.valueOf(body);
-    }
+    return body == null ? def : Boolean.valueOf(body);
   }
 
   public Integer getIntBody() {
     return getIntBody(null);
   }
 
   public Integer getIntBody(Integer def) {
-    if (body == null) {
-      return def;
-    } else {
-      return Integer.parseInt(body);
-    }
+    return body == null ? def : Integer.valueOf(body);
   }
 
   public Long getLongBody() {
     return getLongBody(null);
   }
 
   public Long getLongBody(Long def) {
-    if (body == null) {
-      return def;
-    } else {
-      return Long.parseLong(body);
-    }
+    return body == null ? def : Long.parseLong(body);
   }
 
   public Double getDoubleBody() {
     return getDoubleBody(null);
   }
 
   public Double getDoubleBody(Double def) {
-    if (body == null) {
-      return def;
-    } else {
-      return Double.parseDouble(body);
-    }
+    return body == null ? def : Double.parseDouble(body);
   }
 
   public Float getFloatBody() {
     return getFloatBody(null);
   }
 
   public Float getFloatBody(Float def) {
-    if (body == null) {
-      return def;
-    } else {
-      return Float.parseFloat(body);
-    }
+    return body == null ? def : Float.parseFloat(body);
```

Comment:
Same as above. Please use the same method (i.e. `valueOf` or `parseFloat`) in both `getFloatBody()` and `getFloatAttribute()`.